proxy/src/context/parquet.rs

```diff
@@ -86,6 +86,8 @@ struct RequestData {
     project: Option<String>,
     branch: Option<String>,
     error: Option<&'static str>,
+    success: bool,
+    duration_us: u64,
```

Comment:
Maybe, `total_duration`?

Reply:
maybe worth adding a comment that this is a connection duration, e.g. time from receiving a new socket until proxy pass; otherwise can be mixed up with connection life duration

Reply:
> Maybe, total_duration?

name was agreed in slack with @oruen

> maybe worth adding a comment that this is a connection duration, i.e. time from receiving a new socket until proxy pass; otherwise it can be mixed up with connection life duration

sure